packages/trader/package.json

```diff
@@ -90,7 +90,7 @@
         "@cloudflare/stream-react": "^1.9.1",
         "@deriv-com/analytics": "1.8.0",
         "@deriv-com/quill-tokens": "^2.0.4",
-        "@deriv-com/quill-ui": "^1.11.4",
+        "@deriv-com/quill-ui": "^1.13.9",
```

Comment:
@yauheni-deriv  Please kindly remove the caret ^ from quill-ui version in all package.jsons where it's installed🙏🏻
Please refer to a comment from another PR where are removing it to fully control the version updates from now on: https://github.com/binary-com/deriv-app/pull/15828#discussion_r1670271736